core/sql/exp/ExpHbaseInterface.h

```diff
@@ -62,22 +62,6 @@ class ExHbaseAccessStats;
 
 Int64 getTransactionIDFromContext();
 
-using namespace apache::thrift;
-using namespace apache::thrift::protocol;
-using namespace apache::thrift::transport;
-
-using namespace apache::hadoop::hbase::thrift;
-
-namespace {
-
-  typedef std::vector<std::string> StrVec;
```

Comment:
Yes. We can do this as a different PR because I suspect it would involve more code changes.